apps/src/gamelab/GameLab.js

```diff
@@ -91,6 +101,23 @@ GameLab.prototype.init = function (config) {
     }
   };
 
+  // Override p5.loadImage so we can modify the URL path param
+  if (!GameLab.baseP5loadImage) {
+    GameLab.baseP5loadImage = window.p5.prototype.loadImage;
+    window.p5.prototype.loadImage = function (path, successCallback, failureCallback) {
```

Comment:
yes, `loadImage` is called from lots of places inside p5, including all of the animation code.

yes, I want to wrap the whole thing in an interface once we've figured out what we need to do to it in total.